routes/discordactions.js

```diff
@@ -16,6 +18,7 @@ const router = express.Router();
 router.post("/groups", authenticate, checkIsVerifiedDiscord, validateGroupRoleBody, createGroupRole);
 router.get("/groups", authenticate, checkIsVerifiedDiscord, getAllGroupRoles);
 router.post("/roles", authenticate, checkIsVerifiedDiscord, validateMemberRoleBody, addGroupRoleToMember);
+router.post("/nickname", authenticate, authorizeRoles([SUPERUSER]), checkIsVerifiedDiscord, changeNicknameOfUsers);
```

Comment:
Please change the the API name it shouldn't be /discord-actions/nickname

please use nouns and identify which resource is being updated discord-actions is not a resource

for better API naming please read:https://swagger.io/resources/articles/best-practices-in-api-design/